.github/workflows/x-create-releases.yml

```diff
@@ -0,0 +1,38 @@
+name: X Create releases
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+      prerelease:
+        required: false
+        type: boolean
+        default: true
+    secrets:
+      GH_ORG:
+        required: true
+      GH_TOKEN:
+        required: true
+jobs:
+  create-releases:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    strategy:
+      matrix:
+        repository:
+          - keycloak
+          - keycloak-documentation
+          - keycloak-nodejs-admin-client
+    steps:
+      - name: Get releases
+        id: get-releases
+        run: echo ::set-output name=releases::$(gh release list --repo ${{ secrets.GH_ORG }}/${{ matrix.repository }})
```

Comment:
Listing releases is paginated, that may be an issue. Might be better to check if the specific release exists instead of looking for it in the list. Something like:

`gh release view 0.8111.0 && echo EXIST || echo DOESN'T EXIST`